packages/api-specification/api/application.ts

```diff
@@ -0,0 +1,33 @@
+export interface ApplicationAPI {

```

Comment:
I think we can keep it as is for the time being, and have this as a single ContainerJS application.